packages/core/src/types/config/source.ts

```diff
@@ -61,7 +61,7 @@ export interface SourceConfig {
   /**
    * Used to import the code and style of the component library on demand.
    */
-  transformImport?: false | TransformImport[];
+  transformImport?: ConfigChain<TransformImport[]> | false;
```

Comment:
We can not use `ConfigChain` here, because `TransformImport[][]` is unexpected.

```suggestion
type TransformImportFn = (
  imports: TransformImport[],
) => TransformImport[] | void;

  transformImport?: 
    | TransformImportFn
    | Array<TransformImport | TransformImportFn>
    | false;;
```